lib/services/networkManagement2/package.json

```diff
@@ -6,7 +6,7 @@
     "Rajendran, Deepak <derajen@microsoft.com>",
     "Choubey, Ankur <anchoube@microsoft.com>"
   ],
-  "version": "1.3.0-preview",
+  "version": "1.4.0-preview",
```

Comment:
I ran test with new API and everything worked except recent `app-gw ssl-policy list-available` command due to SDK method being renamed. I addressed this issue in Xplat CLI PR that's going to use this version of SDK.
Should this be considered breaking change?